### ISSUE TITLE
Video Results

### ISSUE THREAD

**Issue by rohaantahir** [Gsak3l/Sports-Analysis-Software#4]
Hi @Gsak3l  great work. Can you share your detected results with me.

**Comment by Gsak3l** [comment]
Thanks!
Do you need the mp4 video output or the txt file?

**Comment by Gsak3l** [comment]
[There you go](https://drive.google.com/file/d/1bm1QSZjZmZbXhlPl704meCo1bN428t4H/view?usp=sharing)

**Comment by rohaantahir** [comment]
Thanks @Gsak3l can you please share the pretrained yolo model files ? to do testing at my end

**Comment by rohaantahir** [comment]
Thanks a lot. Could you please share the annotated dataset in which the model is trained ?

**Comment by rohaantahir** [comment]
I wanted to retrain the model at my end. could please do me this favor ?

**Comment by rohaantahir** [comment]
Hi @Gsak3l  can you please share the command you are using to train the model from scratch I am trying to do it but facing some issues.

**Comment by Gsak3l** [comment]
Navigate to the ./player_detection/yolov5

On this folder create a _custom.yaml_ file with the following code format and a _train_data_ folder
```
# YOLOv5 🚀 by Ultralytics, GPL-3.0 license
# COCO128 dataset https://www.kaggle.com/ultralytics/coco128 (first 128 images from COCO train2017)
# Example usage: python train.py --data coco128.yaml
# parent
# ├── yolov5
# └── datasets
#     └── coco128  ← downloads here


# Train/val/test sets as 1) dir: path/to/imgs, 2) file: path/to/imgs.txt, or 3) list: [path/to/imgs1, path/to/imgs2, ..]
train: train_data/images/train  # train images (relative to 'path') 128 images
val: train_data/images/val # val images (relative to 'path') 128 images
# test:  # test images (optional)

# Classes
nc: 2  # number of classes
# Class names
names: [ 'center', 'MyTeam' ]
```

To train your own dataset you need 4 different subfolders inside the _train_data_ folder like shown in the image

![image](https://user-images.githubusercontent.com/23295116/177086159-03d73cdc-2f3b-45d9-990d-2369f37ac974.png)

The images folder contains both train and validation images for your dataset and the folder labels contains .txt files that point to your objects inside the images.

Then use the command `python train.py --img 640 --batch 16 --epochs 3 --data custom.yaml --weights yolov5s.pt`

Adjust the epochs and batch number to suit your needs. 
From my study, the more epochs you add the less significant the batch value is.
Check yolov5 [#2337](https://github.com/ultralytics/yolov5/issues/2377) for more info
